<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Szerokość:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Wysokość:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="minWidth">
          <property name="toolTip">
           <string>Jeżeli jeden z wymiarów obrazka jest mniejszy niż podana liczba to obrazek nie jest porównywany</string>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="minHeight">
          <property name="toolTip">
           <string>Jeżeli jeden z wymiarów obrazka jest mniejszy niż podana liczba to obrazek nie jest porównywany</string>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Jeżeli jeden z wymiarów obrazka jest mniejszy niż podana liczba to obrazek nie jest porównywany</string>
        </property>
        <property name="text">
         <string>Minimalne wymiary</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dokładne porównanie:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Szybkie porównanie:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Liczba wątków</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="secondaryThreads">
        <property name="toolTip">
         <string>Operacje porównania treści obciążają głównie procesor. Zalecana liczba wątków jest taka jak liczba rdzeni procesora.</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="primaryThreads">
        <property name="toolTip">
         <string>Operacje szybkiego porównania obciążają głównie dysk twardy, a nie procesor. Zalecana liczba wątków: 1</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Porównanie treści</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Rozszerzenia:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="extensions">
        <property name="toolTip">
         <string>Pliki o takich rozszerzenaich będą sprawdzane. Kolejne rozszerzenia należy dodawać po przecinku.</string>
        </property>
        <property name="text">
         <string>png,jpg,jpeg,gif</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="secondaryType">
        <property name="toolTip">
         <string>Dokładne jest szybsze od bardzo dokładne ale zwiększa ryzyko nie wykrycia lub nieprawidłowego wykrycia.</string>
        </property>
        <item>
         <property name="text">
          <string>Dokładne</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bardzo dokładne</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="primarySimultaneously">
      <property name="text">
       <string>Uruchom szybkie porównanie jednocześnie z listowaniem plików</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="secondarySimultaneously">
      <property name="text">
       <string>Uruchom dokładne porównanie jednocześnie z szybkim porównaniem</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>74</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Resetuj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Zapisz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>primaryThreads</tabstop>
  <tabstop>secondaryThreads</tabstop>
  <tabstop>minWidth</tabstop>
  <tabstop>minHeight</tabstop>
  <tabstop>primarySimultaneously</tabstop>
  <tabstop>secondarySimultaneously</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
